frontend/js/components/modals/ModalCreate.vue

```diff
@@ -98,6 +98,7 @@
             if (self.$refs.modal) self.$refs.modal.close()
 
             self.$nextTick(function () {
+              this._isSubmitting = false
```

Comment:
Actually, this needs to happen in case an an error as well.

Reply:
Super good catch! Do you think this could be handled in `finally`? eg.
```
 .finally(() => {
    self.$nextTick(function () {
      this._isSubmitting = false
    })
  })
```

Reply:
Yes, this looks like the best option, thanks!

Reply:
```suggestion
```